template/new-report.hbs

```diff
@@ -16,89 +16,17 @@
-            {{/unless}}
-          </div><!-- .layout-container--short -->
-          <p class="timestamp">{{moment d "MMMM Do YYYY, h:mm:ss a"}}</p>
-        </span>
-      </header><!-- .project__header -->
-    </div><!-- .layout-stacked__header -->
-
-      <div class="layout-stacked__content content_body">
-        <div class="layout-container--short"> <!--create styles -->
-          {{#if paths}}
-          <div class="source-panel">
-            <h1 class="title-content">Scanned the following paths:</h1>
-            <ul>
-              {{#each paths}}<li>{{path}} ({{packageManager}})</li>{{/each}}
-            </ul>
-            <tr class="meta-count"><span>{{summary}}</span></tr>
-          </div>
-          {{/if}}
-          {{#if path}}
-          <div class="source-panel">
-            <h1 class="title-content">Scanned the following path:</h1>
-            <li>{{path}} ({{packageManager}})</li>
-            <tr class="meta-count"><span>{{summary}}</span></tr>
-          </div>
-          {{/if}}
-          
-          <table class="metatable">
-            <tbody>
-              <tr class="meta-row"><th class="meta-row-label">Vulnerabilities</th> <td class="meta-row-value">{{uniqueCount}}</td></tr>
-              <tr class="meta-row"><th class="meta-row-label">Dependencies</th> <td class="meta-row-value">{{dependencyCount}}</td></tr>
-              <tr class="meta-row"><th class="meta-row-label">Taken by CLI</th></tr>
-              <tr class="meta-row"><th class="meta-row-label">Created on</th> <td class="meta-row-value">{{moment d "MMMM Do YYYY, h:mm:ss a"}}</td></tr>
-              <tr class="meta-row"><th class="meta-row-label">Tested with</th> <td class="meta-row-value">{{packageManager}}</td></tr>
-              <tr class="meta-row"><th class="meta-row-label">Repository</th> <td class="meta-row-value">{{path}}</td></tr>
-              <tr class="meta-row"><th class="meta-row-label">Branch</th> <td class="meta-row-value">{{path}}</td></tr>
-              <tr class="meta-row"><th class="meta-row-label">Manifest</th> <td class="meta-row-value">{{packageManager}}</td></tr>
-            </tbody>
-          </table><!-- .meta-counts -->
-        </div>
-
-        <div class="layout-container--short" style="padding-top: 35px;">
-          <div class="cards--vuln">
-            <h2>Remediation advice</h2>
-            {{> actionable-remediations}}
-          </div><!-- cards -->
-        </div>
-      </div><!-- .layout-container -->
-
-      <div class="layout-stacked__content">
-        <div class="layout-container--short" style="padding-top: 35px;">
-          <div class="cards--vuln">
-            <h2>No remediation</h2>
-            {{#each unresolved.vulnerabilities}}
-            {{> vuln-card showSummaryOnly=../showSummaryOnly}}
-            {{/each}}
-          </div><!-- cards -->
-        </div>
-      </div><!-- .layout-container -->
+    {{> header }}
```

Comment:
@ekbsnyk if this merges before #82, we will need to accept these changes in merge conflict

Reply:
This one is a chore, was intended to go in before so final release has all the fancy new things

Reply:
^^ sounds good